tests/e2e_operations.py

```diff
@@ -38,48 +38,78 @@
 from loguru import logger as LOG
 
 
-def force_become_primary(network, args, node):
-    # Ensure all nodes are equally up-to-date
+def force_become_primary(network, args, target_node):
     network.wait_for_node_commit_sync()
-    p, _ = network.find_primary()
-    if p != node:
-        sleep_time = args.election_timeout_ms / 1000
-        LOG.info(
-            f"Suspending {node.node_id} and sleeping {sleep_time}s to trigger election"
-        )
-        # Suspend the target so they trigger an election on resume
-        # This is not guaranteed - if they process any queued message on wakeup, they may avoid calling an election. So try several times
-        attempts = 5
-        for attempt in range(attempts):
-            LOG.info(f"Attempt {attempt} / {attempts}")
+
+    # On each iteration suspend all nodes, and restart just the target node and a quorum of backups
+    # This will have a 1/quorum size chance of winning the election each time
+    # (this can probably be fiddled with to be more consistent)
+    def try_make_target_primary():
+        nodes = network.get_joined_nodes()
+        p, _ = network.find_primary()
+        if p == target_node:
+            return p
+
+        for node in network.get_joined_nodes():
             node.suspend()
-            time.sleep(sleep_time)
-            node.resume()
 
-            try:
-                primary, _ = network.wait_for_new_primary_in(
-                    {node.node_id}, timeout_multiplier=3
-                )
-                break
-            except (infra.network.PrimaryNotFound, TimeoutError) as e:
-                LOG.error(f"No primary found: {e}")
-                continue
+        # sleep for some of the election timeout to drain message queue of target
+        time.sleep(0.1 * args.election_timeout_ms / 1000)
+        target_node.resume()
+        time.sleep(0.1 * args.election_timeout_ms / 1000)
+        target_node.suspend()
+        time.sleep(0.8 * args.election_timeout_ms / 1000)
+
+        target_node.resume()
 
-        assert primary == node
+        quorum_size = (len(nodes) // 2) + 1
+        backup_quorum = [n for n in nodes if n != p and n != target_node][
+            : quorum_size - 1
+        ]
+        for node in backup_quorum:
+            node.resume()
+
+        try:
+            possible_primaries = {n.node_id for n in backup_quorum + [target_node]}
+            primary, _ = network.wait_for_new_primary_in(possible_primaries)
+        except (infra.network.PrimaryNotFound, TimeoutError) as e:
+            LOG.error(f"No primary found: {e}")
+            primary = None
+        for node in nodes:
+            if node.suspended:
+                node.resume()
+        return primary
+
+    iterations = 0
+    eventual_timeout = args.election_timeout_ms / 1000 * 30
```

Comment:
```suggestion
    timeout_multiplier = 30
    eventual_timeout = args.election_timeout_ms / 1000 * timeout_multiplier
```

Naming for consistency with similar constants elsewhere.